src/components/PartnerLogos.js

```diff
@@ -69,29 +81,47 @@ const styles = theme => ({
 
 function PartnerLogos({ classes }) {
   return (
-    <Grid container justify="center" align="center" className={classes.root}>
-      <Grid item>
-        <div className={classes.leftLogo}>
-          <img
-            src={worldbankgroup}
-            alt="World Bank Group"
-            className={classes.img}
-          />
-        </div>
-      </Grid>
-      <Grid item>
-        <div className={classes.centerLogo}>
-          <img src={liquidtelcom} alt="Liquid Telcom" className={classes.img} />
-        </div>
+    <Grid container xs={12}>
+      <Grid
+        container
+        xs={12}
+        direction="row"
+        justify="center"
+        alignItems="center"
+        className={classes.title}
+      >
+        <Typography variant="h5">OUT PARTNERS</Typography>
```

Comment:
**OUR** instead of **OUT**

Reply:
If we're going with centred heading, then lets make it the same size as other headings such as Showcase, The Issues, and Sensors Stories.

Alternative design is Dominion country partners:

![screenshot_2019-03-05 zeplin - project](https://user-images.githubusercontent.com/1779590/53794880-de820d80-3f41-11e9-8158-572d703e6f9f.png)

Reply:
@kilemensi The dominion design is what I originally had before the feedback changes. I'll change the heading instead to match the overall design of sensors.AFRICA

Reply:
@kilemensi Fixed